### ISSUE TITLE
Train

### ISSUE THREAD

**Issue by lyumingzhi** [3dpose/3D-Multi-Person-Pose#5]
Hi thank you for the great job! May I know whether you have a plan to release the training codes and documents for this project?

**Comment by 3dpose** [comment]
Thanks for your interest in our work!

As this repository is intended to provide the inferencing code to replicate the results reported [in the paper](https://arxiv.org/pdf/2104.01797v2.pdf), we did not plan to release the training code. However, as the code of all major components of our method is available under lib/models, you can implement the training code to train on your own data if interested.

**Comment by JosephKKim** [comment]
Thanks for the release of the code. It seems like no code for the loss function.. So you have no plans for releasing loss function?
